test/ctia/test_helpers/core.clj

```diff
@@ -440,10 +440,6 @@
 (defn fixture-max-spec [node-to-spec ns]
   (fixture-spec (fu/require-all node-to-spec) ns))
 
-(defn fixture-fast-gen [t]
-  (with-redefs [gen/vector cgc/vector]
-    (t)))
-
```

Comment:
This is deprecated and predictably broke test.check as I tried to debug this PR. See https://github.com/threatgrid/ctim/pull/277.